en/includes/guides/branding/customize-sms-templates.md

```diff
@@ -2,6 +2,28 @@
 
 The following topics explain how you can customize SMS notifications that are sent to users of your {{ product_name }} organization.
 
+{% if product_name == "Asgardeo" %}
+
+!!! note "SMS branding for B2B applications"
+
+    If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured,
+    note that the sms branding you configure for your root organization also applies to your organizations.
+
+{% else %}
+
+!!! note "SMS branding for B2B applications"
```

Comment:
Is this doc completely not available in IS-7.0.0? Is that the case to not having IS version specific config

Reply:
Yes this is not available in IS 7.0.0